pkgs/tools/dotool/default.nix

```diff
@@ -27,13 +28,15 @@ buildGoModule rec {
   '';
 
   buildInputs = [ libxkbcommon ];
-  nativeBuildInputs = [ pkg-config ];
+  nativeBuildInputs = [ pkg-config scdoc ];
 
   ldflags = [ "-s" "-w" "-X main.Version=${version}" ];
 
   postInstall = ''
-    mkdir -p $out/bin
+    mkdir -p $out/{bin,share/man/man1}
     cp ./dotoold ./dotoolc $out/bin
+    scdoc < doc/dotool.1.scd > dotool.1
+    cp ./dotool.1 $out/share/man/man1/
```

Comment:
Please use `installManPage` helper function from `installShellFiles`

Reply:
Thank you, updated it!